tests/integ/sagemaker/jumpstart/estimator/test_jumpstart_estimator.py

```diff
@@ -0,0 +1,77 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+import os
+import time
+from sagemaker.jumpstart.constants import JUMPSTART_DEFAULT_REGION_NAME
+
+from sagemaker.jumpstart.estimator import JumpStartEstimator
+from tests.integ.sagemaker.jumpstart.constants import (
+    ENV_VAR_JUMPSTART_SDK_TEST_SUITE_ID,
+    JUMPSTART_TAG,
+)
+from tests.integ.sagemaker.jumpstart.utils import (
+    get_sm_session,
+    get_training_dataset_for_model_and_version,
+)
+
+from sagemaker.jumpstart.utils import get_jumpstart_content_bucket
+
+
+MAX_INIT_TIME_SECONDS = 5
+
+
+def test_jumpstart_estimator(setup):
+
+    model_id, model_version = "huggingface-spc-bert-base-cased", "*"
+
+    estimator = JumpStartEstimator(
+        model_id=model_id,
+        role=get_sm_session().get_caller_identity_arn(),
+        sagemaker_session=get_sm_session(),
+        tags=[{"Key": JUMPSTART_TAG, "Value": os.environ[ENV_VAR_JUMPSTART_SDK_TEST_SUITE_ID]}],
+    )
+
+    estimator.fit(
```

Comment:
which instance will this use for training and inference below? concern is do we need limit increases for CI accounts.

Reply:
Good question, this one is ml.p3.2xlarge